src/pattern.jl

```diff
@@ -166,7 +166,7 @@ function connectivity_pattern_to_mat(
 end
 
 function connectivity_pattern_to_mat(
-    xt::AbstractArray{D}, yt::AbstractArray{<:Number}
+    xt::AbstractArray{D}, yt::AbstractArray{<:Real}
 ) where {P,T<:ConnectivityTracer,D<:Dual{P,T}}
```

Comment:
Pattern methods need support for `Complex{T}`.

Reply:
#102